synapse/storage/data_stores/main/events_worker.py

```diff
@@ -189,21 +190,23 @@ def get_events(
     @defer.inlineCallbacks
     def get_events_as_list(
         self,
-        event_ids,
-        check_redacted=True,
-        get_prev_content=False,
-        allow_rejected=False,
+        event_ids,  # type: List[Any]
```

Comment:
```suggestion
        event_ids,  # type: List[str]
```

presumably?

Reply:
also, annotations are a thing:
```suggestion
        event_ids: List[str],
```

Reply:
Is this possible in python 3.5?